src/Backend/Form/DataTransferObject/CheckboxEnabledFieldDataTransferObject.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Backend\Form\DataTransferObject;
+
+class CheckboxEnabledFieldDataTransferObject
+{
+    public bool $enableField = false;
+
+    public string $field = '';
+
+    public static function create(bool $enableField, string $field): self
+    {
+        $dto = new self();
```

Comment:
not a fan of this syntax, either create a private constructor or just use the constructor